packages/flutter_tools/lib/src/commands/generate_localizations.dart

```diff
@@ -224,13 +219,10 @@ class GenerateLocalizationsCommand extends FlutterCommand {
 
   @override
   Future<FlutterCommandResult> runCommand() async {
-    final List<String> outputFileList;
-    File? untranslatedMessagesFile;
-
-    bool format = boolArg('format');
-
+    // Get all options associated with gen-l10n.
+    LocalizationOptions options;
```

Comment:
This can still be final since there are only two possible branches and it gets assigned before referenced and only once total in both:

```suggestion
    final LocalizationOptions options;
```